SharedPackages/BrowserServicesKit/Sources/BrowserServicesKit/PrivacyConfig/Features/PrivacyFeature.swift

```diff
@@ -72,6 +72,7 @@ public enum PrivacyFeature: String {
     case setAsDefaultAndAddToDock
     case contentScopeExperiments
     case extendedOnboarding
+    case tabCrashRecovery
```

Comment:
I’m sneaking this in :) https://github.com/duckduckgo/privacy-configuration/pull/3044
I’ve tested it locally using the auto-generated config: https://duckduckgo.github.io/privacy-configuration/pr-3044/v4/macos-config.json